pkg/api/grpc/manager/manager.go

```diff
@@ -49,13 +49,13 @@ type ConnCreatorFn = func() (grpc.ClientConnInterface, error)
 
 // AppChannelConfig contains the configuration for the app channel.
 type AppChannelConfig struct {
-	Port                 int
-	MaxConcurrency       int
-	TracingSpec          config.TracingSpec
-	EnableTLS            bool
-	MaxRequestBodySizeMB int
-	ReadBufferSizeKB     int
-	BaseAddress          string
+	Port               int
+	MaxConcurrency     int
+	TracingSpec        config.TracingSpec
+	EnableTLS          bool
+	MaxRequestBodySize int // In bytes
+	ReadBufferSize     int
```

Comment:
Please add comment "// In bytes" for all or not, keep same style.

Reply:
Thanks